src/Kafka/Experimental/CreateKafkaInstance/components/FieldCloudRegion.tsx

```diff
@@ -0,0 +1,68 @@
+import { FormGroup, HelperText, HelperTextItem } from "@patternfly/react-core";
+import { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { CloudRegionProps, CloudRegionSelect } from "./CloudRegionsSelect";
+
+export type FieldCloudRegionProps = {
+  validity: "valid" | "required";
+} & Omit<CloudRegionProps, "validated" | "placeholderText">;
+export const FieldCloudRegion: VoidFunctionComponent<FieldCloudRegionProps> = ({
+  value,
+  regions,
+  onChange,
+  isDisabled,
+  validity,
+}) => {
+  const { t } = useTranslation("create-kafka-instance-exp");
+
+  const allRegionsUnavailable = regions.every(
+    ({ isDisabled }) => isDisabled === true
+  );
+
+  const disableControl =
+    isDisabled || allRegionsUnavailable || regions.length === 0;
+
+  const someRegionsUnavailable = regions.some(
+    ({ isDisabled }) => isDisabled === true
+  );
+
+  const validation =
+    validity !== "valid" || allRegionsUnavailable ? "error" : "default";
+
+  const placeholder = allRegionsUnavailable
+    ? t("regions_temporarily_unavailable")
+    : t("select_region");
+
+  const helperTextInvalid = allRegionsUnavailable
+    ? t("all_region_unavailable_helper_text")
+    : t("common:required");
+
+  return (
+    <FormGroup
+      data-testid="cloudRegion"
+      label={t("cloud_region")}
+      fieldId="form-cloud-region-option"
+      isRequired
+      validated={validation}
+      helperText={
+        someRegionsUnavailable ? (
+          <HelperText>
+            <HelperTextItem variant="warning" hasIcon>
+              {t("some_region_unavailable_helper_text")}
+            </HelperTextItem>
+          </HelperText>
```

Comment:
After we reviewed this together, I realized that helper text provided by the form group component includes xs margin-top. But when adding the helper text using the helper text component, those styles aren't getting applied. 

You could include the class `pf-c-form__helper-text` on the `<HelperText>` component to provide additional margin-top. Or the utility class would work, too (until they update the form group component to use a different size): `pf-u-mt-xs`